_pytest/config.py

```diff
@@ -70,6 +70,18 @@ class UsageError(Exception):
     """ error in pytest usage or invocation"""
 
 
+def filename(path):
```

Comment:
How about naming this something which makes it more explicit that it is a argparse type validator? How about `filename_arg` or something like this?